app/models/suggested_councillor.rb

```diff
@@ -2,4 +2,6 @@ class SuggestedCouncillor < ActiveRecord::Base
   has_one :authority, through: :councillor_contribution
   belongs_to :councillor_contribution
   validates :councillor_contribution, :name, :email, presence: true
+  validates_email_format_of :email, :message => "format seems not right.
+  Does it follow the format of example@email.com?", :unless => Proc.new{|c| c.email.blank?}
```

Comment:
@equivalentideas before this, it was displaying multiple error messages when the email was submitted blank ("Email cannot be blank" and ""format seems not right. Does it follow the format of example@email.com?").